baselines/exo.md

```diff
@@ -1,34 +1,36 @@
+**`TLP:CLEAR`**
+
 # CISA M365 Security Configuration Baseline for Exchange Online
 
-Microsoft Exchange Online provides users easy access to their email and supports organizational meetings, contacts, and calendars. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Exchange Online security.
+Microsoft Exchange Online gives users easy access to their email and supports organizational meetings, contacts, and calendars. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Exchange Online security.
```

Comment:
Microsoft 365 Exchange Online is a cloud-based messaging platform that delivers email, calendar, contacts, and tasks.